google-cloud-clients/google-cloud-contrib/google-cloud-spanner-jdbc/src/test/java/com/google/cloud/spanner/jdbc/ITAbstractSpannerTest.java

```diff
@@ -211,7 +211,11 @@ public ITConnection createConnection(
   public ITConnection createConnection(
       List<StatementExecutionInterceptor> interceptors,
       List<TransactionRetryListener> transactionRetryListeners) {
-    StringBuilder url = new StringBuilder("cloudspanner:/").append(getDatabase().getId().getName());
+    StringBuilder url = new StringBuilder("cloudspanner:");
+    if (getTestEnv().getTestHelper().getOptions().getHost() != null) {
+      url.append(getTestEnv().getTestHelper().getOptions().getHost().replace("https:", ""));
```

Comment:
Sorry just noticed this - should this be made consistent with the change we just merged for ITAbstractJdbcTest, or better yet - can we pull out this into a helper method?